bmc/redfish_local.go

```diff
@@ -182,3 +182,23 @@ func (r RedfishLocalBMC) GetManager() (*Manager, error) {
 
 	return nil, err
 }
+
+func (r *RedfishLocalBMC) GetBootOrder(systemUUID string) ([]string, error) {
```

Comment:
Would it make sense to also implement the BIOS and boot order settings for the local provider? That way we could add unit tests against the BMC emulator and have at least some test coverage here.

Reply:
sure I am not done. forgot to mark this as a draft

Reply:
@afritzler I made a proposal to change the bmclient. now you would only need to implement the interface methods if they differ from the default `RedfishBMCClient`. if not overwritten the methods of `RedfishBMCClient` are being called.